src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  align-items: center;
+  padding: 23px 0;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  line-height: normal;
+  color: #000;
+  transition: color 0.3s;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+a.is-active {
```

Comment:
Remove the tag selector here

Reply:
sorry since this comment was made not to the specific row but to a few I thought I had to switch the class for the tag as well 

![Screenshot_2023-07-28-18-10-37-54_4641ebc0df1485bf6b47ebd018b5ee76](https://github.com/mate-academy/layout_moyo-header/assets/140621438/18538112-c340-4fe2-8019-97b4948b35e0)